src/configuration.ts

```diff
@@ -112,14 +112,14 @@ export function findConfiguration(configFile: string | null, inputFilePath: stri
  * @returns An absolute path to a tslint.json file
  * or undefined if neither can be found.
  */
-export function findConfigurationPath(suppliedConfigFilePath: string | null, inputFilePath: string) {
+export function findConfigurationPath(suppliedConfigFilePath: string | null, inputFilePath?: string) {
```

Comment:
same as above

Reply:
@ajafff I tried adding that here too but then I run into a strange compiler error that I can't really make sense of:

```typescript
src/configuration.ts(95,58): error TS2345: Argument of type 'string | undefined' is not assignable to parameter of type 'string'.
  Type 'undefined' is not assignable to type 'string'.
error Command failed with exit code 2.
```

From what I understand, this _should_ work but the TS compiler is ignoring the other overload? Not sure. Any ideas? Thanks!